lib/view_model/active_record/controller.rb

```diff
@@ -91,8 +96,34 @@ def prerender_viewmodel(...)
 
   private
 
+  # Viewmodels allow for integer or uuid ids
+  class ViewmodelIdSerializer < IknowParams::Serializer
+    def initialize
+      super(::Object)
+    end
+
+    def load(val)
+      case val
+      when ::Integer
+        val
+      when ::String
+        if IknowParams::Serializer::UUID.matches_type?(val)
+          IknowParams::Serializer::UUID.load(val)
+        else
+          IknowParams::Serializer::Integer.load(val)
+        end
+      else
+        raise IknowParams::Serializer::LoadError.new(
+                "Incorrect type for #{self.class.name}: #{val.inspect}:#{val.class.name}")
+      end
+    end
+
+    set_singleton!
+    json_value!
+  end
+
   def viewmodel_id
-    parse_param(:id)
+    parse_param(:id, with: ViewmodelIdSerializer)
```

Comment:
Do we explicitly support fetch by multiple id? We have clients that make use of this, and either it works because the backend doesn't enforce types and thus viewmodel_id is already sometimes an array, or it doesn't work and the clients somehow don't hit that call path.

I think show _should_ support fetch by multiple id. Perhaps you can do so by copy/pasting the `viewmodel_ids` parser from `delete`?

Reply:
We explicitly support `index` filtered by multiple id, by having a default `:id` array-typed request filter defined. I think that's a more natural place for it than introducing a new route "show, but show for specific set".